### ISSUE TITLE
Correctly show next backup task in menu

### ISSUE THREAD

**Issue by m3nu** [borgbase/vorta#1099]
Currently it shows the time of the oldest profile with scheduled backups, not next schedule. Needs correct sorting.

Example: next backup is on 16:xx, but menu shows 18:00

![Screen Shot 2021-11-06 at 16 27 22](https://user-images.githubusercontent.com/3916435/140609583-ff1cc578-46ac-4f6e-a638-eb40166b347e.jpg)

**Comment by m3nu** [comment]
Also: after hibernation (and missing the previous backup) it just says "Nothing scheduled" until the backup timer wakes up.

**Comment by jayeclark** [comment]
@m3nu I'm a new contributor, but I found the issue & have a fix for it ... I just need to verify a couple more things tomorrow morning and then will move the PR out of draft mode. 

I did want to mention that the tray menu only displays the _time_ of the next scheduled backup, which can be misleading. For example, a job scheduled for 12 AM three days in the future will display as 00:00 which, as a user, I would generally interpret as meaning the next job will run the next time it is 12 AM. (Not 12 AM several days in the future) 

I'm a new contributor so I'm not sure if this would be a common situation or if the typical use case is to have backups run so frequently that the misunderstanding would rarely arise.
